### ISSUE TITLE
Screen Size Spoofing does not work [correctly] 

### ISSUE THREAD

**Issue by jahnson** [sereneblue/chameleon#30]
First of all, thank you for reviving this addon.  Most addons that spoof your client only spoof the user agent string, which is not enough.  There are many other parameters that can be used to fingerprint you, so a more comprehensive addon is necessary.  

Under "Options", "Script Injection Options", I have selected "Enable Script Injection".  Then, For "Screen size spoofing" I have selected 1920x1080 (a very common screen size).  
Then when I go to 
https://whoer.net/#extended
it reports that both my screen and window size are 1430 x 1760, which happens to be the window size at this moment.  That differs from the setting I chose.  

[the same screen size is shown when I use http://mybrowserinfo.com/detail.asp]

**Comment by sereneblue** [comment]
This is similar to these issues: https://github.com/sereneblue/chameleon/issues/25 and https://github.com/sereneblue/chameleon/issues/19.

I'll have to do some more research to see what's the best way to go about solving this.

**Comment by danielcra** [comment]
See also here: https://arthuredelstein.github.io/tordemos/media-query-fingerprint.html
It can read me screen size when spoofing is on.

**Comment by Thorin-Oakenpants** [comment]
https://demos.traudt.xyz/css/media/index.html is a prettier PoC :) and I think sereneblue is right, that you can't spoof this

I deal with it by using uMatrix in extreme hard mode, i block all css except 1st party by default. Not many sites (for me) require 3rd party css (or if it is, it's one I can set globally such as wordpress)

**Comment by Bumbadawg** [comment]
@Thorin-Oakenpants good idea, although it might still be impossible to block those media when 1st party. 
Therefore it might be a good idea to have script that parses those @media tags, remove any HTTP requests in them, or at least notify about them, clean them by modifying it if needed.
Does such script / webext exist ?
